### PR TITLE
Pass onAuthChange callback into embed

### PR DESCRIPTION
**Pull request by jde**
## What does this PR do?

Allows the host page to provide a callback that fires on the parent page when the Talk auth state changes. 

## How do I test this PR?

- embed a talk stream with an onAuthChange callback:
```
<div id="coral_talk_8938213609030934"></div>
<script src="http://localhost:5000/embed.js" async onload="
  Coral.Talk.render(document.getElementById('coral_talk_8938213609030934'), {
    talk: 'http://localhost:5000',
    onAuthChange: function(auth) {
      console.log(auth);
    }
  });
"></script>
```
- log in through Talk
- see the auth state logged in the console from the parent page

### CODE REVIEW

File: client/coral-framework/actions/auth.js [hunk 1]
```diff
@@ -315,6 +316,9 @@ export const checkLogin = () => (dispatch) => {
     .catch((error) => {
       console.error(error);
       dispatch(checkLoginFailure(`${error.translation_key}`));
+    })
+    .then(() => {
+      pym.sendMessage('checkLogin', JSON.stringify(getState().auth));
```
Comment (alinhle) [inline client/coral-framework/actions/auth.js]: Why not passing the result of the rest call? The redux state of `auth` currently also contains state of the frontend interface related to auth, which I don't think we should expose.

**Comment by alinhle** [comment]
What’s the use case for this?

**Comment by meddy** [comment]
Hello @cvle , the use case is dynamically showing content based on the auth state of the embed app.

**Comment by jde** [comment]
Hey @meddy, sorry for the delay. This is good to go.

Before we can merge your branch into master you will need to update your branch with our master. Please let us know when you are doing this so we can coordinate accepting the PR.

Thanks!

**Review by alinhle — APPROVED** [review]

**Comment by alinhle** [comment]
Awesome @meddy, thanks for your contribution!

**Pull request merged**